tests/unit/test_work_item_pages.py

```
@@ -0,0 +1,134 @@
+    def work_item(self, client: PlaneClient, workspace_slug: str, project: Project):
+        """Create a work item and delete it after the test."""
+        wi = client.work_items.create(
+            workspace_slug,
+            project.id,
+            CreateWorkItem(name=f"Test WI for pages {int(time.time())}"),
+        )
+        yield wi
+        try:
+            client.work_items.delete(workspace_slug, project.id, wi.id)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)